packages/ipfs/src/core/runtime/libp2p-browser.js

```diff
@@ -35,13 +35,13 @@ module.exports = () => {
     config: {
       peerDiscovery: {
         autoDial: true,
+        // Optimization
+        // Requiring bootstrap inline in components/libp2p to reduce the cli execution time
```

Comment:
This is browser config, the reference to cli execution time isn't relevant here.